src/components/Elements/ListItem.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { View, Text, TouchableOpacity } from 'react-native';
 import Icon from 'react-native-vector-icons/MaterialCommunityIcons';
 import styles from './styles/itemStyles';
-import card from '../styles/card';
+import Card from './Card';
```

Comment:
Card must be uppercase as JS is case sensitive

Reply:
I thought it should be like this: import styles from './styles/itemStyles';

Reply:
Styles is fine. Check "Card"